src/components/test/calculate.test.js

```diff
@@ -40,21 +43,7 @@ describe('test calculation', () => {
     expect(calculate(sampleInputData, '=')).toEqual(expectedOutput);
   });
 
-  it('Should return 10', () => {
```

Comment:
- [ ] All features and scenarios of the `operate.js` and `calculate.js` modules should be covered using Jest. Be sure to write multiplication test (operation === 'x') and modulo (operation === '%') using Jest. ✍🏻